### ISSUE TITLE
there is (unofficial) NetBSD/pkgsrc support

### ISSUE THREAD

**Issue by ghost** [maralla/pack#35]
Not really an issue, but I can confirm that at least unofficially NetBSD is working, I have packaged it for pkgsrc (currently it is in pkgsrc-wip/pack), and a simple install, list, uninstall action for 1 vim package worked.

Since pkgsrc targets more than 1 platform, this might show where it is not supported (in addition to Windows) once it is merged into pkgsrc. So depending on what your scope of "supported" in the README description for this project is, you could mention this or leave it be.
